Uppsala/UppEt20.xml

```diff
@@ -0,0 +1,524 @@
+                                    <title type="complete" ref="LIT1960Mashaf#AnaphoraEpiphanius"></title>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.7.10">
+                                    <locus from="135rb" to="144rb"/>
+                                    <title type="complete" ref="LIT1960Mashaf#AnaphoraJacobSerug"></title>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.7.11">
+                                    <locus from="144rb" to="153vb"/>
+                                    <title type="complete" ref="LIT1960Mashaf#AnaphoraCyril"></title>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.7.12">
+                                    <locus from="153vb" to="161va"/>
+                                    <title type="complete" ref="LIT1960Mashaf#AnaphoraFirstGregoryNyssa"></title>
+                                    <note>Here attributed to <persName ref="PRS4825Gregory"/>.</note>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.7.13">
+                                    <locus from="161vb" to="164rb"/>
+                                    <title type="complete" ref="LIT1960Mashaf#AnaphoraGregoryNathanael"></title>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.7.14">
+                                    <locus from="164va" to="178ra"/>
+                                    <title type="complete" ref="LIT1960Mashaf#AnaphoraBasil"></title>
+                                </msItem>
+                                
+                            </msItem>
+
+
+                            <msItem xml:id="ms_i1.8">
+                                <locus from="178ra" to="190rb"/>
+                                <title type="complete">Prayers, mostly prayers of incense (<foreign xml:lang="gez">ṣalota ʿəṭān</foreign>)</title>
+                                
+                            <msItem xml:id="ms_i1.8.1">
+                                <locus from="178ra"/>
+                                <title type="complete"></title>
+                                <incipit>
+                                    <hi rend="rubric">ሰላም፡ እምኀበ፡ <surplus>እምኀበ፡</surplus> እግዚአ</hi>ብሔር፡ አብ፡ ወልደ፡ ዋሕድ፡ 
+                                    ዘተፈነወ፡ ወኃደረ፡ ውስተ፡ ማኅፀና፡ <hi rend="rubric">ለማርያም፡</hi> ድንግል፡
+                                </incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.8.2">
+                                <locus from="178vb"/>
+                                <title type="complete">Prayer of incense</title>
```

Comment:
all different from LIT2252salota?

Reply:
It seems indeed that LIT2252salota (and it's individual parts) corresponds to many parts of this collection of prayers. Thank you for the identification! I think I will add incipits in LIT2252salota (based on one of the printed editions already quoted there), to facilitate identification in the future, if that is ok.

Reply:
(And I will revise LIT1960Mashaf to include a reference to LIT2252salota as well)

Reply:
Thank you!